src/main/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsController.java

```diff
@@ -56,4 +60,17 @@ public JobsController(JobsService jobsService, AuthService authService) {
 
         return noContent().build();
     }
+
+    @GetMapping(path = "")
+    @ApiOperation("Get list of jobs for a service")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Success")
+    })
+    public List<Job> getAll(
```

Comment:
list results should be wrapped in a model https://hmcts.github.io/restful-api-standards/

Reply:
It should. left for next PR but can do properly initially

Reply:
fair enough